starter-code/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <title>Blog</title>
   <!-- TODO: Add a meta tag to properly set the viewport and initial scale -->
-
+  <meta name="viewport" content="initial-scale=2">
```

Comment:
Nice! Note: adding this content="width=device-width does make your life a bit easier